### PR TITLE
chain/yoda: Add commands for improve experience of starting yoda process

### PR DESCRIPTION
**Pull request by taobun**
## Implementation details
- Add `multi-send` command in the same level of send cmd to send coin from 1 account to multiple accounts with the same amount.
- Add `add-reporters` command to pack `MsgAddReport`s in one transaction.
- Add flag in `yoda keys list` to get only address using in `multi-send` and `add-reporters` command. Look example at `start_yoda.sh`.

Please ensure the following requirements are met before submitting a pull request:
---
- [x] The pull request is targeted against the correct target branch
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests. (Test by run chain can request and report correctly)
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: chain/x/oracle/client/cli/tx.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
 		GetCmdActivate(cdc),
 		GetCmdAddReporter(cdc),
 		GetCmdRemoveReporter(cdc),
+		GetCmdAddReporters(cdc),
```
Comment (sorawit) [inline chain/x/oracle/client/cli/tx.go:49]: I don't think we need a new command. Let's change the current `add-reporter` to take any number of reporters.

File: chain/x/oracle/client/cli/tx.go [hunk 2]
```diff
@@ -559,3 +560,45 @@ $ %s tx oracle remove-reporter band1p40yh3zkmhcv0ecqp3mcazy83sa57rgjp07dun --fro
 
 	return cmd
 }
+
+// GetCmdAddReporter implements the add reporter command handler.
+func GetCmdAddReporters(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-reporters [reporter1] [reporter2] ...",
+		Short: "Add an agent authorized to submit report transactions.",
```
Comment (sorawit) [inline chain/x/oracle/client/cli/tx.go]: an agent -> agents

File: chain/x/oracle/client/cli/tx.go [hunk 3]
```diff
@@ -559,3 +560,45 @@ $ %s tx oracle remove-reporter band1p40yh3zkmhcv0ecqp3mcazy83sa57rgjp07dun --fro
 
 	return cmd
 }
+
+// GetCmdAddReporter implements the add reporter command handler.
```
Comment (sorawit) [inline chain/x/oracle/client/cli/tx.go]: add reporters

File: chain/cmd/bandcli/multi_send.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+			// parse coins trying to be sent
+			coins, err := sdk.ParseCoins(args[0])
+			if err != nil {
+				return err
+			}
+			msg := bank.NewMsgMultiSend(
+				[]bank.Input{bank.NewInput(cliCtx.GetFromAddress(), mulCoins(coins, int64(len(args[1:]))))},
+				[]bank.Output{},
+			)
+			for _, arg := range args[1:] {
```
Comment (sorawit) [inline chain/cmd/bandcli/multi_send.go]: I think it's safer to use Coin.Add. So we start with `inputCoins = nil` and do `inputCoins = inputCoins.Add(coins)` in the loop and contract `bank.Input` after we complete the loop. That way we don't need `mulCoins` function too.

File: chain/cmd/bandcli/multi_send.go [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+	cmd := &cobra.Command{
+		Use:   "multi-send [amount] [to_address1] [to_address2] ....",
+		Short: "Create and sign a multi-send tx",
+		Args:  cobra.MinimumNArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+
+			// parse coins trying to be sent
```
Comment (sorawit) [inline chain/cmd/bandcli/multi_send.go]: // Parse the coins we are trying to send

File: chain/cmd/bandcli/multi_send.go [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+func mulCoins(coins sdk.Coins, multiplier int64) sdk.Coins {
+	var newCoins sdk.Coins
+	for _, coin := range coins {
+		newCoins = append(newCoins, sdk.NewCoin(coin.Denom, coin.Amount.MulRaw(multiplier)))
+	}
+	return newCoins
+}
+
+// MultiSendTxCmd will create a multi-send tx and sign it with the given key.
```
Comment (sorawit) [inline chain/cmd/bandcli/multi_send.go]: MultiSendTxCmd creates ... and signs ...

**Review by sorawit — CHANGES_REQUESTED** [review]
Looking good. Some comments from me.

**Review by sorawit — APPROVED** [review]
Looks great!

**Review by tansawit — APPROVED** [review]

**Pull request merged**